bin/nuxt-dev

```diff
@@ -111,7 +111,7 @@ function startDev(oldInstance) {
         if (oldInstance && oldInstance.builder) {
           return oldInstance.builder.unwatch()
         } else {
-          return nuxt.listen(port, host)
+          return nuxt.listen(port, host, nuxtConfig.https)
```

Comment:
Maybe unnecessary to pass `https`, config can be got through `this.options` inside `nuxt.js`